README.md

```diff
@@ -62,6 +62,22 @@ public static Intent intentForDeepLinkMethod(Context context) {
 }
 ```
 
+
+You can also annotate static methods that take a `Context` and return a `TaskStackBuilder`. `DeepLinkDispatch` will call that
+method to create `Intent` from `TaskStackBuilder` last `Intent` and use it when starting your Activity via that registered deep link:
+
+```java
+@DeepLink("http://example.com/deepLink/{id}/{name}")
+  public static TaskStackBuilder intentForTaskStackBuilderMethods(Context context) {
```

Comment:
`TaskStackBuilder` requires API 16 and newer. You should mention that in the README too